components/Data_1/index.js

```diff
@@ -5,7 +5,7 @@ import moment from "moment";
 const now = moment();
 const lockedTime = now.set().format("HH:mm");
 
-export default function InformationPage() {
+export default function DataObject_1() {
```

Comment:
I am not sure whether this is a better way to name your component: I would argue that you won't know next week what `DataObject_1` does in comparison to `DataObject_2`.

You can use `Information` and the English word for "Marke", that's not a problem at all.